libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -5,18 +5,25 @@
 # Usage:
 #   get_object_files_for_test(<result var>
 #                             <skipped_entrypoints_var>
+#                             <internal_obj>
 #                             <target0> [<target1> ...])
 #
 #   The list of object files is collected in <result_var>.
 #   If skipped entrypoints were found, then <skipped_entrypoints_var> is
 #   set to a true value.
 #   targetN is either an "add_entrypoint_target" target or an
 #   "add_object_library" target.
-function(get_object_files_for_test result skipped_entrypoints_list)
+#   If internal_obj is TRUE, then we collect `target.__internal__` for entry points.
+function(get_object_files_for_test result skipped_entrypoints_list internal_obj)
```

Comment:
Could we use `cmake_parse_arguments` here?
It is not clear what `TRUE` (or `FALSE`) means when invoking the function
```
get_object_files_for_test(link_object_files skipped_entrypoints_list TRUE ${fq_deps_list})
```

Reply:
Done.